crates/nargo/Cargo.toml

```diff
@@ -22,7 +22,6 @@ aztec_wasm_backend = { optional = true, package = "barretenberg_wasm", git = "ht
 marlin_arkworks_backend = { optional = true, git = "https://github.com/noir-lang/marlin_arkworks_backend", rev = "144378edad821bfaa52bf2cacca8ecc87514a4fc" }
 
 [features]
-default = ["plonk_bn254"]
```

Comment:
@TomAFrench Why did you remove this default? Now no backend will be built for nargo without using a feature flag.

Reply:
This is because to be able to build `nargo_cli` for different backends require us to turn off this feature flag. We'd then have to by default turn off this default feature in the workspace so we might as well remove it.

Reply:
I think we need to leave a default because building from source (for an end-user) shouldn't **require** a feature flag. If you want to remove the default feature, we should be falling back to the backend instead of requiring one